Live/Classes/Food.h

```diff
@@ -0,0 +1,18 @@
+#ifndef __FOOD__H__
+#define __FOOD__H__
```

Comment:
hard to be consistent with header guards, consider using `#pragma once` instead.

Reply:
Done